src/legacy/core_plugins/kibana/public/discover/doc_table/components/table_row.js

```diff
@@ -109,7 +109,7 @@ module.directive('kbnTableRow', function ($compile, $httpParamSerializer, kbnUrl
       };
 
       $scope.getContextAppHref = () => {
-        const path = kbnUrl.eval('#/context/{{ indexPattern }}/{{ anchorType }}/{{ anchorId }}', {
+        const path = kbnUrl.eval('#/context/{{ indexPattern }}/{{ anchorId }}', {
           anchorId: $scope.row._id,
           anchorType: $scope.row._type,
```

Comment:
Looks like this row can also go.

Reply:
Thanks for your hints, turned out, much more can be cleanup up, removed the anchorType from a bunch of files, mainly the functional tests